requirements/developer.txt

```diff
@@ -13,4 +13,5 @@ pygit2==1.6.1
 pyspelling
 pre-commit
 twine
+onnxruntime
```

Comment:
it should be dynamically loaded based on model runtime type.

Reply:
I think we should do that for our common dependencies but for our developer dependencies we should install everything so we can run tests in CI